### ISSUE TITLE
Modified configuration files backup fails

### ISSUE THREAD

**Issue by grenagit** [YunoHost/issues#2184]
### Describe the bug

Since 3 days manually modified configuration files backup fails!

### Context

- Hardware: *computer at home*
- YunoHost version: 11.1.16 (stable)
- I have access to my server: *Through SSH | through the webadmin*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*

### To reproduce

```sh
sudo yunohost backup create --system conf_manually_modified_files --debug
```

### Logs

https://paste.yunohost.org/raw/ugunecovoy

**Comment by grenagit** [comment]
May be fixed by [helpers: fix previous change about using YNH_APP_ACTION ... which is … ](https://github.com/YunoHost/yunohost/commit/8c25aa9b9faaf190792738277f71d74822f11088)?

**Comment by alexAubin** [comment]
Yes, should be fixed in 11.1.17
